packages/blade/src/components/Counter/_decisions/decisions.md

```diff
@@ -0,0 +1,46 @@
+# Counter 
+
+Counters are visual indicators that contains numerical values, tallies or counts in regards to some context.
```

Comment:
how are we thinking to tackle use cases where we might want to show the counters on top of some icon or something?
<img width="70" alt="Screenshot 2022-10-26 at 1 23 30 PM" src="https://user-images.githubusercontent.com/11384858/197967307-b2bd5c67-93d2-4b65-a68c-2b05cb38dbbb.png">

Reply:
I don't think there's any opinion in this from design side, so right now we don't support it out of the box. So users have to use absolute positioning. 

I don't see any mention about this usecase in figma also. 

// cc @saurav12

Reply:
🤔 

"Counters can't be combined with icons". 

<img width="558" alt="image" src="https://user-images.githubusercontent.com/35374649/197970821-1af0996c-5d1d-416d-92ca-f45b2e514ee0.png">

Reply:
This usecase of showing an indicator is designed in [indicators](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=8505%3A55)

Reply:
We might tackle the use-case that @kamleshchandnani mentioned for a counter on top of the icon. I see it as a valid use-case.